ci-operator/step-registry/telcov10n/slcm/telcov10n-slcm-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: telcov10n-slcm
+  steps:
+    # pre:
+    # - ref: telco5g-jobs-router
+    # - ref: telco5g-cluster-setup
+    test:
+    - ref: telcov10n-slcm-tests
+    env:
+      ANSIBLE_SKIP_TAGS: NOT_REQUIRED
+      CNF_IMAGE: NOT_REQUIRED
+      DCI_PIPELINE_FILES: du-deployment.yml
```

Comment:
We can add the env vars in the config, that way we can have different jobs using the same workflow.
For example, workflows for 4.12 and 4.14.
See example [here](https://github.com/openshift/release/pull/51506)

Reply:
Where is the config? Sorry it's nearly impossible what I have to do from the example link.

Reply:
Do you mean here? https://github.com/openshift/release/blob/6b14a15eb0f0724674e0db7af6b7e30b2ee29ae0/ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__far-edge-vran-ptp-4.15.yaml#L23

Reply:
Yes, some of the vars can be in the step registry but injecting the OCP version for example in the config will allow you to create multiple jobs using the same workflow

Reply:
Done